resources/api-extract.json

```diff
@@ -439,4 +439,4 @@
     "sap.zen.commons.layout.PositionContainer": "control",
     "sap.zen.dsh.ValueHelpDialog": "content"
   }
-}
+}
```

Comment:
last empty line missing. is this desired?

Reply:
Generated by script, so I wouldn't bother if the CI is fine

Reply:
Done 👌